examples/thirdweb/components/NFTDropCard/NFTDropCard.tsx

```diff
@@ -61,7 +61,7 @@ type Props = {
   chainId?: string
   showMedia?: boolean
   showDescription?: boolean
-  showClaimed?: boolean
+  totalClaimed?: string
```

Comment:
I would recommend using more strict type than `string`.

```suggestion
  totalClaimed?: 'nototal' | 'total' | 'max' | 'available'
```

And perhaps even better would be to use an enum instead of string values

Reply:
Good suggestion let me add it tomorrow.